lib/logproto/logproto-text-server.c

```diff
@@ -153,12 +155,12 @@ log_proto_text_server_get_raw_size_of_buffer(LogProtoTextServer *self, const guc
   if (self->convert_scale)
     return g_utf8_strlen((gchar *) buffer, buffer_len) * self->convert_scale;
 
-  if (self->reverse_buffer_len < buffer_len * 6)
+  if (self->reverse_buffer_len < requried_reverse_buffer_length)
     {
```

Comment:
typo in the name of the variable, but also, I'd probably extract the reallocation routine instead of extracting this to a separate
value.

For instance if you had log_proto_text_server_maybe_realloc_reverse_buffer(buffer_len * 6) with a comment about the "6" or that 6 extracted into a separate value, this whole thing would be even more easy to read.